test/integration/dockerregistryclient_test.go

```diff
@@ -224,14 +222,19 @@ func doTestRegistryClientImage(t *testing.T, registry, version string) {
 	}
 }
 
+func TestRegistryClientImageV2ManifestV2(t *testing.T) {
+	t.Skip("openshift/schema-v2-test-repo is being pushed by Docker 1.10, which causes this test to fail because digests don't match")
```

Comment:
By the way, there's already a helper to get docker version: https://github.com/miminar/origin/blob/remote-layer-federation/test/extended/images/helper.go#L326.